packages/massa-web3/src/experimental/basicElements/address.ts

```diff
@@ -4,11 +4,12 @@ import { Version, Versioner } from '../crypto/interfaces/versioner'
 import VarintVersioner from '../crypto/varintVersioner'
 import { checkPrefix, extractData } from './internal'
 import { PublicKey } from './keys'
+import varint from 'varint'
 
 const ADDRESS_PREFIX = 'A'
 const ADDRESS_USER_PREFIX = 'U'
 const ADDRESS_CONTRACT_PREFIX = 'S'
-
+export const BLAKE3_HASH_BYTELENGTH = 32
```

Comment:
This name is not well chosen as it exposes the underlying logic of the Address object's functioning. You should either rename it or change the logic (as proposed below) to encapsulate this information.